docs/master/performance/query-caching.md

```diff
@@ -0,0 +1,8 @@
+# Query caching
+
+In order to speed up the GraphQL query parsing, its results can be stored in the Laravel cache.
+
+Query caching is enabled in non-local environments by default. You can define cache store and cache duration,
+see `config/lighthouse.php`.
+
+The GraphQL schema doesn't affect query parsing, so you have to flush query cache only on the package update.
```

Comment:
```suggestion
Make sure to flush the query cache when you deploy an upgraded version of the dependency `webonyx/graphql-php`:

    some-one-liner that clears the cache?
```

Reply:
I can add another command like `lighthouse:query:cache:clear` but it only clears the cache store. So I guess it's easier to just use the laravel default command.

Reply:
That Laravel command clears all caches?

Reply:
it fully clears the default store. If the user specified non-default store for query cache they should add an argument to the command
`php artisan cache:clear --store=query_cache_store`